code/API_definitions/device-reachability-status.yaml

```diff
@@ -0,0 +1,397 @@
+openapi: 3.0.3
+info:
+  title: Device Reachability Status Retrieval
+  description: |
+    This API provides the customer with the ability to query device:
+      - Reachability status

```

Comment:
This list suggests that other infos might come later, but this is a specialized API.